### PR TITLE
Fixed #3806: SAXParser supported with manual binding in JSB

### PR DESCRIPTION
**Pull request by pandamicro**

### CODE REVIEW

File: cocos/scripting/javascript/bindings/cocos2d_specifics.hpp [hunk 1]
```diff
@@ -200,5 +201,44 @@ class JSTouchDelegate: public cocos2d::Object
...
+            pInstance = new __JSSAXDelegator();
+        }
+		return pInstance;
+	};
+    
+    ~__JSSAXDelegator();
+    
+    cocos2d::SAXParser* getParser();
+    
+    std::string parse(std::string path);
```
Comment (dumganhar) [inline cocos/scripting/javascript/bindings/cocos2d_specifics.hpp]: Please use `const std::string&` for parameters.

File: cocos/scripting/javascript/bindings/cocos2d_specifics.hpp [hunk 2]
```diff
@@ -200,5 +201,44 @@ class JSTouchDelegate: public cocos2d::Object
...
+    
+    bool preloadPlist(std::string path) {
+        return true;
+    }
```
Comment (dumganhar) [inline cocos/scripting/javascript/bindings/cocos2d_specifics.hpp:223]: What 's  this empty implementation used for?

Comment (pandamicro) [inline cocos/scripting/javascript/bindings/cocos2d_specifics.hpp:223]: In html5 version, SAXParser stores a list of preload in SAXParser, it's useless in c++, so it's just for binding the function

File: cocos/scripting/javascript/bindings/cocos2d_specifics.hpp (hunk 1 above)
Comment (pandamicro) [inline cocos/scripting/javascript/bindings/cocos2d_specifics.hpp]: ok

**Pull request merged**